articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -22,78 +22,90 @@ ms.custom: aaddev
+   - Select a key duration **In 1 year**.
+   - The key value will display when you select **Add**.
+   - Copy the value of the key for later. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible from the Azure portal.
 >
 > [!div class="sxs-lookup" renderon="portal"]
-> #### Step 1: Configure your application in the Azure portal
-> For the code sample for this quickstart to work, you need to:
-> 1. Add reply URLs as `http://localhost:8080/msal4jsamples/secure/aad` and `https://localhost:8080/msal4jsamples/graph/users`.
-> 1. Create a Client Secret.
-> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
-> > [Make this change for me]()
->
-> > [!div id="appconfigured" class="alert alert-info"]
-> > ![Already configured](media/quickstart-v2-aspnet-webapp/green-check.png) Your application is configured with these attributes.
+
+#### Step 1: Configure your application in the Azure portal
+
+For the code sample for this quickstart to work, you need to:
+
+1. Add reply URLs as `http://localhost:8080/msal4jsamples/secure/aad` and `https://localhost:8080/msal4jsamples/graph/me`.
+1. Create a Client Secret.
+
+ > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+ > [Make this change for me]()
+
+ > [!div id="appconfigured" class="alert alert-info"]
+ > ![Already configured](media/quickstart-v2-aspnet-webapp/green-check.png) Your application is configured with these attributes.
 
 #### Step 2: Download the code sample
- 
+
  [Download the Code Sample](https://github.com/Azure-Samples/ms-identity-java-webapp/archive/master.zip)
- 
- #### Step 3: Configure the code sample 
- 
+
+#### Step 3: Configure the code sample
+
  1. Extract the zip file to a local folder.
  1. If you use an integrated development environment, open the sample in your favorite IDE (optional).
- 1. Open the **application.properties** file, which can be found in *src/main/resources/*.
- 1. Replace application properties.
-   1. Find `aad.clientId` and update the value of `Enter_the_Application_Id_here` with the **Application (client) ID** value of the application you registered. 
-   1. Find `aad.authority` and update the value of `Enter_the_Tenant_Name_Here` with the **Directory (tenant) ID** value of the application you registered.
-   1. Find `aad.secretKey` and update the value of `Enter_the_Client_Secret_Here` with the **Client Secret** you created in **Certificates & Secrets** for the application you registered.
+
+ 1. Open the application.properties file, which can be found in src/main/resources/ folder and replace the value of the fields *aad.clientId*, *aad.authority* and *aad.secretKey* with the respective values of **Application Id**, **Tenant Id** and **Client Secret** as the following:
+
+    ```file
+    aad.clientId=Enter_the_Application_Id_here
+    aad.authority=https://login.microsoftonline.com/Enter_the_Tenant_Id_Here/
```

Comment:
I think the string should be "Enter_the_Tenant_Name_Here" to be consistent with other quickstarts..